### PR TITLE
Native image bash launcher version and vm config

### PR DESCRIPTION
**Pull request by zakkak**
As discussed in https://github.com/graalvm/mandrel/issues/230 and https://github.com/oracle/graal/issues/3313 in 21.1-dev the `native-image` bash-launcher no longer prints a meaningful version.

This PR makes the bash launcher print:

```
native-image 21.1.0 Java 11 CE (Java Version 11.0.11+5-jvmci-21.1-b02)
```

for GraalVM CE and

```
native-image 21.1.0.0 Java 11 Mandrel Distribution (Java Version 11.0.10+9)
```

for Mandrel.

The corresponding **native** `native-image` binaries print:

```
GraalVM 21.1.0 Java 11 CE (Java Version 11.0.11+5-jvmci-21.1-b02)
```

for GraalVM CE and

```
GraalVM 21.1.0.0 Java 11 Mandrel Distribution (Java Version 11.0.10+9)
```

for Mandrel.

### CODE REVIEW

**Comment by zakkak** [comment]
Hi @olpaw, can you please review this?

We would also like this to get backported to `release/graal-vm/21.1` if possible.

Thanks

**Review by olpaw — APPROVED** [review]
LGTM

Glad we found something that works for us and and also for you.

File: substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/DefaultOptionHandler.java [hunk 1]
```diff
@@ -81,7 +81,12 @@ public boolean consume(ArgumentQueue args) {
             case "--version":
                 args.poll();
                 singleArgumentCheck(args, headArg);
-                String message = System.getProperty("java.vm.version");
+                String message;
+                if (NativeImage.IS_AOT) {
+                    message = System.getProperty("java.vm.version");
+                } else {
+                    message = "native-image " + NativeImage.graalvmVersion + " " + System.getProperty("org.graalvm.config", "CE");
```
Comment (pejovica) [inline substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/DefaultOptionHandler.java:88]: You could use `NativeImage.graalvmConfig` here.
```suggestion
                    message = "native-image " + NativeImage.graalvmVersion + " " + NativeImage.graalvmConfig;
```

**Review by pejovica — APPROVED** [review]

File: substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/DefaultOptionHandler.java (hunk 1 above)
Comment (zakkak) [reply substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/DefaultOptionHandler.java]: Fixed, thanks!

**Comment by olpaw** [comment]
Your PR just merged on master (should be visible in a few minutes). I also created a PR against 21.1 and it should get included there as well.

**Comment by zakkak** [comment]
Thank you Paul!

**Comment by olpaw** [comment]
> Your PR just merged on master (should be visible in a few minutes). I also created a PR against 21.1 and it should get included there as well.

The change also got included in the release branch: https://github.com/oracle/graal/commits/release/graal-vm/21.1

**Pull request merged**